### PR TITLE
[dcd] update bacon sparse configs

### PR DESCRIPTION
**Pull request by yacuken**

### CODE REVIEW

File: sparse/etc/acdbdata [hunk 1]
```diff
@@ -0,0 +1 @@
+/system/etc/acdbdata
```
Comment (sledges) [inline sparse/etc/acdbdata:1]: Could you please rename this commit message, saying why acdbdata symlink is needed?

File: patterns/jolla-hw-adaptation-bacon.yaml [hunk 2]
```diff
@@ -73,9 +73,8 @@ Requires:
...
 - pattern:jolla-sailfish-cellular-apps
 - telepathy-ring
-#- pulseaudio-modules-droid-glue
+- pulseaudio-modules-droid-glue
 
 # For devices where test_gps eventually gets a fix:
-- geoclue-provider-hybris
-
+#- geoclue-provider-hybris-community
 Summary: Jolla HW Adaptation bacon
```
Comment (sledges) [inline patterns/jolla-hw-adaptation-bacon.yaml:80]: Could you squash this commit with its original one into none please? :)

File: sparse/etc/acdbdata (hunk 1 above)
Comment (yacuken) [reply sparse/etc/acdbdata:1]: If I understand right it's firmwares for sound dsp.
Vendor library acdbloader try to load files from /etc

Comment (sledges) [reply sparse/etc/acdbdata:1]: perfect, so you could rename the commit to "[dcd] Fix firmware find path for sound dsp"

Comment (yacuken) [reply sparse/etc/acdbdata:1]: ok. edit.
also I delete redundant commits (pulseaudio-modules-droid-glue remove and rever) as mal- advise

**Comment by sledges** [comment]
:heart: LGTM

**Pull request merged**